tslint.json

```diff
@@ -4,19 +4,13 @@
     "tslint-config-prettier"
   ],
   "rulesDirectory": [
-    "node_modules/codelyzer",
-    "node_modules/rxjs-tslint"
+    "node_modules/codelyzer"
   ],
   "rules": {
     "ordered-imports": true,
     "arrow-return-shorthand": true,
     "callable-types": true,
     "class-name": true,
-    "no-unused-variable": true,
```

Comment:
Intentional deletions?

Reply:
hmm...was deprecated a while ago.  didn't realize we still had it. I guess Angular finally auto-removed it

Reply:
as for rxjs-tslint and stuff. I manually removed those because its original purpose was to aid with the new rxjs version migration.  We're quite a few major rxjs version ahead of that migration and rxjs-tslint is no longer compatible